images/build/cross/Dockerfile

```diff
@@ -80,7 +62,7 @@ RUN echo "deb http://archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/
   && apt-get install -y build-essential \
   && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do apt-get install -y crossbuild-essential-${platform}; done
 
-ARG PROTOBUF_VERSION=3.0.2
+ARG PROTOBUF_VERSION=3.9.1
```

Comment:
Is this this version it was before? Or should this be 3.0.2?

Reply:
@cblecker -- Reverted that bump. I was attempting to bump protobuf to what's in k/k@`master`, but it doesn't look as simple as I assumed.